.travis.yml

```diff
@@ -4,7 +4,5 @@ node_js:
   - "10"
   - "8"
   - "6"
-  - "4"
   - "0.12"
   - "0.10"
-  - "0.8"
```

Comment:
While I think 0.8 will work with this, getting the tests to run is hard.

Reply:
The problem with node 0.8 is that the lib relies on the cookies lib:

```
npm ERR! Error: No compatible version found: cookies@'^0.7.1'
npm ERR! Valid install targets:
npm ERR! ["0.1.0","0.1.1","0.1.2","0.1.3","0.1.4","0.1.5","0.1.6","0.2.0","0.2.1","0.2.2","0.2.3","0.3.0","0.3.1","0.3.2","0.3.3","0.3.4","0.3.5","0.3.6","0.3.7","0.3.8","0.4.0","0.4.1","0.5.0","0.5.1","0.6.0","0.6.1","0.6.2","0.7.0","0.7.1","0.7.2","0.7.3"]
npm ERR!     at installTargetsError (/home/travis/.nvm/v0.8.28/lib/node_modules/npm/lib/cache.js:719:10)
npm ERR!     at /home/travis/.nvm/v0.8.28/lib/node_modules/npm/lib/cache.js:641:10
npm ERR!     at saved (/home/travis/.nvm/v0.8.28/lib/node_modules/npm/node_modules/npm-registry-client/lib/get.js:138:7)
npm ERR!     at Object.oncomplete (fs.js:302:15)
npm ERR! If you need help, you may report this log at:
npm ERR!     <http://github.com/isaacs/npm/issues>
npm ERR! or email it to:
npm ERR!     <npm-@googlegroups.com>
```

Reply:
We can take 0.8 as a separate PR if we want to make it work.